client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx

```diff
@@ -139,6 +139,23 @@ class AutoRenewDisablingDialog extends Component {
 							'%(expiryDate)s is a date string, e.g. May 14, 2020',
 					}
 				);
+			default:
+				return translate(
+					'By canceling auto-renewal, your %(productName)s plan for %(siteDomain)s will expire on %(expiryDate)s. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'By canceling auto-renewal, your %(planName)s plan for %(siteDomain)s will expire on %(expiryDate)s. When it does, you'll lose access to key features you may be using on your site. To avoid that, turn auto-renewal back on or manually renew your plan before the expiration date.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/allow-auto-renew-toggle-always)